lib/yaml_db/serialization_helper.rb

```diff
@@ -184,7 +184,7 @@ def self.each_table_page(table, records_per_page=1000)
         quoted_table_name = Utils.quote_table(table)
 
         (0..pages).to_a.each do |page|
-          query = Arel::Table.new(table, ActiveRecord::Base).order(id).skip(records_per_page*page).take(records_per_page).project(Arel.sql('*'))
```

Comment:
This had been added in dcff7d7b07ef1b68e74412a3c78be81170792da2 but I can't recall why.